Proposals/TypeNetwork_ParametricAxes/ProposalSummary_ytas.md

```diff
@@ -47,6 +47,8 @@ general area that ascenders live in is an attribute that all users can easily po
 may know by name. This name fits into the systematic structure of the overall system of
 proposed axes: Y dimension transparency of ascenders.
 
+ytas is measured from the top of the ytlc to the top of the ascender
```

Comment:
Since this is part of the portion that would be published in the registry, can you please formulate as a full, punctuated sentence; e.g.,

"The ytas value is measured from the x-height to the top of the ascender."

(Note: since the ytlc measure is described in terms of x-height, it seems equivalent and perhaps clearer to use x-height here rather than to "top of ytlc".)

Also, wouldn't this make sense to be added to the Scale Interpretation field, rather than Additional Information?